### ISSUE TITLE
[Feature] Allow StyleBot to style the StyleBot editor

### ISSUE THREAD

**Issue by dannymcgee** [ankit/stylebot#677]
I've tried targeting `.stylebot-app` in my global stylesheet, but the style doesn't get applied (presumably because of the shadow DOM encapsulation). Would be happy to open a PR, just let me know if you would be open to it and if you have any suggestions for the implementation. :)

**Comment by ankit** [comment]
@dannymcgee I will be open to it if you want to open a PR. however, what specifically did you want to change in the style? If it's a generalized change to improve UX, maybe it can be made to the core editor itself

**Comment by mutte-79** [comment]
I would also be interested in this.

It's the font size and a darker editor background color I would customize. 

Other than that I love the plugin and it's my css-editor of choice.

**Comment by dannymcgee** [comment]
Hey, sorry I never followed up on this, just got busy with other things!

Most of what I use StyleBot for is applying my own syntax theme to sites with a lot of code — I just find it easier to read (and nicer to look at :P) if it's in a color palette that I'm familiar with. So it occurred to me one day while using it that it would be really nice if I could use StyleBot to style StyleBot. Although Monaco is kind of a pain to style with raw CSS in my experience, so it might be even better to just support custom Monaco themes.

Anyway, I might circle back around to this eventually, but my plate is pretty full at the moment, so if anybody else gets the urge to tackle this in the meantime don't wait up for me. :)

**Comment by NubeBuster** [comment]
Yeah we need a recursive stylebot-edits-stylebot portal. Be careful not to create a paradox as the simulation will overload and our universe will seize to progress in time.
